src/SketchRNN/index.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* eslint prefer-destructuring: ['error', {AssignmentExpression: {array: false}}] */
+/* eslint no-await-in-loop: 'off' */
+/*
+SketchRNN
+*/
+
+import * as ms from '@magenta/sketch';
+import callCallback from '../utils/callcallback';
+import modelPaths from './models';
+
+const PATH_START_LARGE = 'https://storage.googleapis.com/quickdraw-models/sketchRNN/large_models/';
+const PATH_START_SMALL = 'https://storage.googleapis.com/quickdraw-models/sketchRNN/models/';
+const PATH_END = '.gen.json';
+
+export default class SketchRNN {
```

Comment:
We are following a pattern of not exporting the class directly. ml5 creates as a function that parses init arguments and calls the corresponding class. See: https://github.com/ml5js/ml5-library/blob/master/src/LSTM/index.js#L157

This also allows to create methods without passing `new`. So instead of:
```javascript
var model = new ml5.SketchRNN()
```
just
```javascript
var model = ml5.SketchRNN()
```